[flake8] Fix flake8 errors

vchrombie opened the pull request:
This PR fixes the existing flake8 errors.

vchrombie commented on kidash/kidash.py:
```diff
@@ -1118,8 +1127,8 @@ def get_dashboard_name(panel_file):
     elif kibana:
-        logger.error("Wrong panel format (can't find 'dashboard' or 'index_patterns' fields): %s",
-                     panel_file)
+        logger.error("Wrong panel format (can't find 'dashboard' or "
+                     "'index_patterns' fields): %s", panel_file)
```
instead of such a format, we can adapt a different multiline too like
```python
        logger.error("""Wrong panel format (can't find 'dashboard' or 
                     index_patterns' fields): %s""", panel_file)
```

WDYT?

sduenas replied:
Multiline format will add a new line and tabs at the beginning of the second line, so it will be hard to parse in case we need it.

vchrombie commented on kidash/kidash.py:
```diff
@@ -1588,45 +1636,52 @@ def fetch_dashboard(elastic_url, dash_id, es_index=None):
 
-            with open(export_file, 'w') as f:
-                f.write(json.dumps(dash_json, indent=4, sort_keys=True))
+            with open(export_file, 'w') as fa:
+                fa.write(json.dumps(dash_json, indent=4, sort_keys=True))
```
since `f` is already in use, changing this file object to `fa`

sduenas replied:
Good!

vchrombie commented on kidash/kidash.py:
```diff
@@ -1588,45 +1636,52 @@ def fetch_dashboard(elastic_url, dash_id, es_index=None):
                 index_pattern_importable = {"index_patterns": [index_pattern]}
-                with open(export_file_index, 'w') as f:
-                    f.write(json.dumps(index_pattern_importable, indent=4, sort_keys=True))
+                with open(export_file_index, 'w') as fb:
+                    fb.write(json.dumps(index_pattern_importable, indent=4,
+                                        sort_keys=True))
```
same as above

sduenas replied:
Good!

vchrombie reviewed (commented):
Hi @sduenas 
I had a few suggestions/doubts with this PR, so opening some comments about them.

Feel free to suggest any changes.

vchrombie commented on kidash/kidash.py:
```diff
@@ -789,8 +800,7 @@ def get_index_pattern_json(elastic, index_pattern_id):
 def get_search_from_vis(elastic, vis):
     search_id = None
     vis_json = get_vis_json(elastic, vis)
-    if not vis_json:
-        search_id
```
Also, removed this completely as this seems to have no effect.

sduenas replied:
Makes sense.

sduenas reviewed (commented):
LGTM. Just a minor comment before we merge it. Do you think you can add the same `MANIFEST.in` we have in other repositories such in Perceval? It has some `flake8` configuration that we should include here too.

vchrombie committed:
[flake8] Fix flake8 errors

This commit adds the setup.cfg file which has flake8
configurations for the project. It also fixes the
existing flake8 errors.

This commit also includes the metadata files and tests
in MANIFEST.in file.

Signed-off-by: Venu Vardhan Reddy Tekula <venu@bitergia.com>

sduenas reviewed (approved):
LGTM
